go.mod

```diff
@@ -145,7 +145,7 @@ require (
 	github.com/vmware/goipmi v0.0.0-20181114221114-2333cd82d702
 	github.com/vmware/govmomi v0.29.0
 	github.com/xanzy/go-gitlab v0.74.0
-	github.com/zclconf/go-cty v1.10.0
```

Comment:
this might be dangerous, not sure where/how we use it?
this is the same problem we had with the packer plugin

Reply:
Great catch, removed this update for hcl